core/src/main/scala/Configuration.scala

```diff
@@ -1,16 +1,21 @@
 package com.gu.contentatom.renderer
 
-import com.gu.contentatom.thrift.Atom
+import com.gu.contentatom.renderer.ArticleConfiguration.CommonsDivisionConfiguration
 
 sealed trait Configuration
 
 sealed trait NilConfiguration extends Configuration
 case object NilConfiguration extends NilConfiguration
 
 final case class ArticleConfiguration(
-  ajaxUrl: String
+  ajaxUrl: String,
+  commonsDivisionConfiguration: CommonsDivisionConfiguration = CommonsDivisionConfiguration()
```

Comment:
I'd prefer it for now if the config was completely transparent and there was not default value.

Reply:
Can we use the same spelling as the atom name itself? So just `commonsdivision`, all lowercase?

Reply:
... perhaps in the future properties like `ajaxUrl` should be stuck in a `global` subconfig object.

Reply:
I can't believe I didn't make it all lowercase after all the trouble that has caused previously!